rrmngmnt/ssh.py

```diff
@@ -56,7 +55,7 @@ def __init__(self, executor, timeout=None, use_pkey=False):
             self._ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
             if use_pkey:
                 self.pkey = paramiko.RSAKey.from_private_key_file(
-                    ID_RSA_PRV
+                    ID_RSA_PRV % self._h.user.name
```

Comment:
I think _h is in Host
On Feb 1, 2016 16:09, "Nelly Credi" <notifications@github.com> wrote:

> In rrmngmnt/ssh.py
> <https://github.com/rhevm-qe-automation/python-rrmngmnt/pull/10#discussion_r51421449>
> :
>
> > @@ -56,7 +55,7 @@ def __init__(self, executor, timeout=None, use_pkey=False):
> >              self._ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
> >              if use_pkey:
> >                  self.pkey = paramiko.RSAKey.from_private_key_file(
> > -                    ID_RSA_PRV
> > +                    ID_RSA_PRV % self._h.user.name
>
> os.path.expanduser
>
> and I still cant find _h
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rhevm-qe-automation/python-rrmngmnt/pull/10/files#r51421449>
> .
>